airflow/www/static/js/api/useExtraLinks.ts

```diff
@@ -34,19 +34,22 @@ export default function useExtraLinks({
   executionDate,
   mapIndex,
   extraLinks,
+  tryNumber,
 }: {
   dagId: string;
   taskId: string;
   executionDate: string;
   mapIndex?: number | undefined;
   extraLinks: string[];
+  tryNumber?: number | undefined;
 }) {
   return useQuery(
-    ["extraLinks", dagId, taskId, executionDate, mapIndex],
+    ["extraLinks", dagId, taskId, executionDate, mapIndex, tryNumber],
     async () => {
       const data = await Promise.all(
         extraLinks.map(async (link) => {
           const definedMapIndex = mapIndex ?? -1;
+          const defaultTryNumber = tryNumber ?? 0;
```

Comment:
@bbovenzi , do you know if this is a safe assumption to make when the try number is undefined?

Reply:
@bbovenzi , @pierrejeambrun - can you take a look at this when you have a min?

Reply:
Sorry, I just got back from vacation.

Let's not include try number if it isn't defined